lib/elements/analysis_results_controller.dart

```diff
@@ -90,8 +90,16 @@ class AnalysisResultsController {
 
     final columnElem = DivElement()..classes.add('issue-column');
 
+    final weHaveLineNo = issue.line >= 1;
+    final lineNoInfo = weHaveLineNo ? 'line ${issue.line}' : '';
+    final additionalSourceInfo = (issue.sourceName == 'main.dart')
+        ? ''
+        : '${weHaveLineNo ? ' of ' : ''}${issue.sourceName} ';
+    final locationSeparator =
+        (additionalSourceInfo.isNotEmpty || lineNoInfo.isNotEmpty) ? ' • ' : '';
+
```

Comment:
https://dart.dev/guides/language/effective-dart/design#avoid-abbreviations

```suggestion
    final hasLineNumber = issue.line >= 1;
    final lineNumberInfo = hasLineNumber ? 'line ${issue.line}' : '';
    final additionalSourceInfo = (issue.sourceName == 'main.dart')
        ? ''
        : '${hasLineNumber ? ' of ' : ''}${issue.sourceName} ';
    final locationSeparator =
        (additionalSourceInfo.isNotEmpty || lineNumberInfo.isNotEmpty)
            ? ' • '
            : '';
```